templates/parachain/node/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde = { features = ["derive"], workspace = true, default-features = true }
 jsonrpsee = { version = "0.22", features = ["server"] }
 futures = "0.3.28"
 serde_json = { workspace = true, default-features = true }
+docify = "0.2.8"
```

Comment:
This is a template used by external people. We should clearly not add this docify stuff here.